src/gpu_sysman.c

```diff
@@ -1411,7 +1450,8 @@ static bool gpu_freqs_throttle(gpu_device_t *gpu) {
       ok = false;
       break;
     }
-    if (!set_freq_labels(freqs[i], &metric)) {
+    double dummy;
+    if (!set_freq_labels(freqs[i], &metric, &dummy)) {
```

Comment:
Maybe instead of "dummy" it could be possible to pass NULL here.

Reply:
Yes, that looks nicer.